packages/components/src/components/drawer/drawer.ts

```diff
@@ -281,6 +289,7 @@ export default class SdDrawer extends SolidElement {
   render() {
     return html`
       <div
+        id="drawer"
```

Comment:
Is this `id` and the one on L340 needed?

Reply:
This id is necessary, it is used to add "aria-controls" attibute to the close-button making the relation between the two elements available for a11y as well.
The id on L340 was unnecessary and has been deleted now.